### ISSUE TITLE
Spike: Investigate session token refresh to extend logins

### ISSUE THREAD

**Issue by techsmyth** [alkem-io/client-web#1160]
As a user I want that my tokens are refreshed so that I do not have to login daily.

## Acceptance criteria
- [ ] We know what / how token refresh flows are supported in Kratos and how to manage token expiration
- [ ] Timeboxed to half a day

## Additional information

Desired state:
- [ ] The token is refreshed silently, so that when I go back to the page after being a time away I am still logged in.
- [ ] Refresh token validity duration to be configurable. Default: 7 days?
- [ ] User is not aware of the token being refreshed

The refresh flow is a separate interaction. This is only for Ory Kratos.

**Comment by hero101** [comment]
[Possible solution](https://github.com/ory/kratos/issues/615#issuecomment-667003129)
Waiting on possible solution - [feat: Add session renew capabilities, #615](https://github.com/ory/kratos/pull/2146)

> There are no plans to add any type of "refresh token" (which is an OAuth2 terminology) to Ory Kratos. If you wish to use OAuth2 for mobile authentication in first-party scenarios, the appropriate route would be combining Ory Kratos + Ory Hydra.


> This depends on your security model. If you are a bank, you probably want a 30 minutes time out which is reset on user interaction. If you are a community forum, 90 days is probably also ok.
>
>If you want to have 24 hours with the option to reset the timer on interaction I think it would make most sense to have this as a parameter in the /sessions/whoami endpoint with e.g. /session/whoami?reset_expiry or something similar.

Some threads on the topic
[A perpetual session for mobile app](https://github.com/ory/kratos/discussions/1603)
[Refresh Sessions Without Having to Log In Again](https://github.com/ory/kratos/issues/615)
[Admin/Selfservice session management](https://github.com/ory/kratos/issues/655)

**Comment by techsmyth** [comment]
Ok, so there is a clear fix waiting to be merged into Kratos. I suggest to move this one to blocked, or close this one and create a new one into the blocked column that has the results of this investigation and what we are waiting on.

**Comment by techsmyth** [comment]
Spike is done; follow up captured on: https://github.com/alkem-io/client-web/issues/1785
